tests/fixtures/request_payloads/batch_feature_table.json

```diff
@@ -4,9 +4,9 @@
         "Instead, update the test tests/unit/test_generate_payload_fixtures.py#test_save_payload_fixtures.",
         "Run `pytest --update-fixtures` to update it."
     ],
-    "_id": "646f6c1c0ed28a5271fb02da",
-    "batch_request_table_id": "646f6c1c0ed28a5271fb02d9",
-    "deployment_id": "646f6c1c0ed28a5271fb02d6",
-    "feature_store_id": "646f6c190ed28a5271fb02a1",
+    "_id": "647b5ba9875a4313db21a1df",
+    "batch_request_table_id": "647b5ba9875a4313db21a1de",
```

Comment:
nit: would be nice to be able to "lock" these ID's so that they aren't always regenerated whenever we update fixures

Reply:
Indeed. It makes PRs look intimidating

Reply:
Will revert these fixture changes for now and manually update the new payload fixture. We can refactor in separate PR since this one is rather large atm